python/plugins/processing/tests/testdata/expected/raster_unique_values_report.html

```diff
@@ -0,0 +1,29 @@
+<html><head><meta http-equiv="Content-Type" content="text/html;charset=utf-8"/></head><body>
+<p>Analyzed file: /home/webmaster/dev/cpp/QGIS/python/plugins/processing/tests/testdata/raster.tif (band 1)</p>
+<p>Extent: 270736.0673250681720674,4458888.9563983473926783 : 270899.8544675338780507,4459029.5745217483490705</p>
```

Comment:
It'd be handy to export all these extra properties as outputs from the alg too, so they could be used in later model steps

Reply:
@nyalldawson , you tink so? Which one(s) would you like to be exposed? TOTAL_PIXEL_COUNT, and NODATA_COUNT?

Reply:
Expose ALL the things!

Extent, projection (as authid), all the numeric values

They could all be potentially reused later in a model